Camille/gen/download.ps1

```diff
@@ -6,3 +6,11 @@ $DD = ((Invoke-WebRequest -Uri "http://ddragon.leagueoflegends.com/realms/na.jso
 echo "DD version $DD."
 
 Invoke-WebRequest -Uri "http://ddragon.leagueoflegends.com/cdn/$DD/data/en_US/champion.json" -OutFile .\.champion.json
+
+echo "Downloading static data"
+
+Invoke-WebRequest -Uri "http://static.developer.riotgames.com/docs/lol/seasons.json" -OutFile .\Enums\.seasons.json
+Invoke-WebRequest -Uri "http://static.developer.riotgames.com/docs/lol/queues.json" -OutFile .\Enums\.queues.json
+Invoke-WebRequest -Uri "http://static.developer.riotgames.com/docs/lol/maps.json" -OutFile .\Enums\.maps.json
+Invoke-WebRequest -Uri "http://static.developer.riotgames.com/docs/lol/gameModes.json" -OutFile .\Enums\.gameModes.json
+Invoke-WebRequest -Uri "http://static.developer.riotgames.com/docs/lol/gameTypes.json" -OutFile .\Enums\.gameTypes.json

```

Comment:
Actually `download.ps1` isn't used anymore, I moved the downloads to `Camille.csproj` to be more os-agnostic and seems I forgot to remove this

Reply:
I removed `downlaod.ps1` and moved the added download endpoints to `Camille.csproj`